raiden-ts/src/channels/reducer.ts

```diff
@@ -192,32 +204,14 @@ const channels: Reducer<RaidenState['channels'], RaidenAction> = createReducer(
   .handle(channelClose.success, channelCloseSuccessReducer)
   .handle(channelSettle.success, channelSettleSuccessReducer);
 
-const pendingTxs: Reducer<RaidenState['pendingTxs'], RaidenAction> = (
-  state = initialState.pendingTxs,
-  action: RaidenAction,
-): RaidenState['pendingTxs'] => {
-  // filter out non-ConfirmableActions's
-  if (!isActionOf(ConfirmableActions, action)) return state;
-  // if confirmed==undefined, add action to state
-  else if (action.payload.confirmed === undefined) return [...state, action];
-  // else (either confirmed or removed), remove from state
-  else {
-    const newState = state.filter(
-      (a) => a.type !== action.type || action.payload.txHash !== a.payload.txHash,
-    );
-    if (newState.length !== state.length) return newState;
-    return state;
-  }
-};
-
 /**
  * Nested/combined reducer for channels
- * blockNumber, tokens & channels reducers get its own slice of the state, corresponding to the
+ * blockNumber, tokens & pendingTxs reducers get its own slice of the state, corresponding to the
  * name of the reducer. channels root reducer instead must be handled the complete state instead,
  * so it compose the output with each key/nested/combined state.
  */
-const channelsReducer = partialCombineReducers(
-  { blockNumber, tokens, channels, pendingTxs },
-  initialState,
-);
+const partialReducer = partialCombineReducers({ blockNumber, tokens, pendingTxs }, initialState);
+// channels reducer is a reduce-reducers like reducer
```

Comment:
This sounds like a tongue twister :smile:, can we clarify what a `reduce-reducers like reducer` is in the comment?

Reply:
It's already in the first place this kind of reducer is mentioned: root `raidenReducer`. `reduce-reducers` is a known utility on the redux ecosystem, I just borrowed the idea, but will write a comment to make the information more easily accessible